pom.xml

```diff
@@ -226,6 +226,7 @@
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
                     <includeSystemScope>true</includeSystemScope>
+                    <executable>true</executable>
```

Comment:
I borked my rebase, so it included Marcus's change too. This is the only one included.